.github/conversational/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+**Note: for support questions, please use stackoverflow**. This repository's issues are reserved for feature requests and bug reports.
+
+* **I'm submitting a ...**
+  - [ ] bug report
+  - [ ] feature request
+  - [ ] support request => Please do not submit support request here, see note at the top of this template.
+
+
+* **Do you want to request a *feature* or report a *bug*?**
+
+
+
+* **What is the current behavior?**
+
+
+
+* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem** via
+https://plnkr.co or similar (you can use this template as a starting point: http://plnkr.co/edit/tpl:AvJOMERrnz94ekVua0u5).
```

Comment:
I don't how plunker helps here, other than making things more complicated than they already are. I feel implementing this for a simple static blog generated by Pelican is bit of an overkill. 
Simple - Give an appropriate description of the bug, steps to reproduce it and attach screenshots or use similar means for the same.